src/notifications/NotificationsHandler.tsx

```diff
@@ -293,6 +295,16 @@ export const NotificationsHandler = ({ walletReady }: Props) => {
           });
         }
       }
+    } else if (type === NotificationTypes.walletConnect) {
+      logger.info(
+        `NotificationsHandler: handling wallet connect notification`,
+        { notification }
+      );
+      setHasPendingDeeplinkPendingRedirect(true);
```

Comment:
This is triggered after clicking on a notification i.e. after sending a `sendTransaction` RPC to the wallet.